code/game/gamemodes/wizard/magic_tarot.dm

```diff
@@ -441,6 +462,7 @@
 /datum/tarot/strength
 	name = "XI - Strength"
 	desc = "May your power bring rage."
+	extended_desc = "grants the user strength only matched by powerful vampires. The strength renders the user unable to handle ranged weapons."
```

Comment:
```suggestion
	extended_desc = "grants the user strength beyond belief, but renders them unable to handle ranged weapons."
```
~~Also I probably should've given more damage to gargs with this status effect huh~~